tests/test_libraries.py

```diff
@@ -2878,3 +2878,9 @@ def test_narwhals(pyi_builder):
     pyi_builder.test_source("""
         import narwhals
     """)
+
```

Comment:
Linter requires functions to be separated by two empty lines.

Reply:
Beat me by a second! 😆  
4a1c752